admin_manual/maintenance/manually-moving-data-folders.rst

```diff
@@ -0,0 +1,91 @@
+=====================================
+How To Manually Move a Data Directory
+=====================================
+
+If you need to move your ownCloud data directory from its current location to
+somewhere else, here is a manual process that you can take to make it happen.
+
+.. NOTE:: 
+   This example assumes that:
+
+   - The current folder is: `/var/www/owncloud/data`
+   - The new folder is: `/mnt/owncloud`
+   - You're using Apache as your webserver
+
+1. Stop Apache
+2. Use rsync to sync the files from the current folder to the new one the folders -> 
+3. Create a symbolic link from the new directory to the old one 
+4. Double-check `the directory permissions`_ on the new directory 
+5. Restart Apache
+
+To save time, here's the commands which you can copy and use::
+
+  apachectl -k stop 
+  rsync -avz /var/www/owncloud/data /mnt/owncloud
+  ln -s /mnt/owncloud /www/owncloud/data
+  apachectl -k graceful 
+
+.. NOTE:: 
+   If you're on Centos, try `service httpd stop` and `service httpd start`.
+   If you're on Debian/Ubuntu try `sudo service apache2 stop` and `sudo service apache2 start`
+
+Fix Hardcoded Database Path Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to manually change the location of the data folder in the database,
+run the SQL below:
+
+.. code-block::
+   
+  UPDATE oc_storages SET id='local::/var/www/owncloud/data' 
+    WHERE id='local::/mnt/owncloud'
+
+The other area to check is the `oc_jobs` table. The logrotate process may have
+hard-coded a non-standard (or old) value for the data path. To check it, run the
+SQL below and see if any results are returned:
+
+.. code-block::
+
+  SELECT * FROM oc_jobs WHERE class = 'OC\Log\Rotate';
+
+If any are, run the SQL below to update them, changing the value as appropriate.
+
+.. code-block::
+
+  UPDATE oc_jobs SET argument = '/your/new/data/path' 
```

Comment:
I think it's not that easy. The argument is usually json_encoded so the path appears somewhere in the middle.

You can probably test this by setting the config.php value `log_rotate_size` to some value